src/components/VmDisks/utils.js

```diff
@@ -7,12 +7,5 @@ import { localeCompare } from '_/helpers'
  * Bootable drives sort first, then sorted number aware alphabetically.
```

Comment:
Please change the comment accordingly.

Reply:
@sgratch - I changed the comment according the function behavior.